src/App/App.js

```diff
@@ -37,20 +38,26 @@ class App extends React.Component {
   state = {
     authed: false,
     pendingUser: true,
+    // uid: '',
+    // weather: [],
   }
 
   componentDidMount() {
     connection();
     this.removeListener = firebase.auth().onAuthStateChanged((user) => {
       if (user) {
+        // eslint-disable-next-line prefer-destructuring
+        const uid = user.uid;
```

Comment:
is there a reason that you're not writing this as `const { uid } = user;` ?

Reply:
Updated